dev/src/Docs/DoctumConfigBuilder.php

```diff
@@ -64,7 +64,6 @@ public static function buildConfigForVersion($version)
             'build_dir'            => "$gaxRootDir/tmp_gh-pages/%version%",
             'cache_dir'            => "$gaxRootDir/cache/%version%",
             'remote_repository'    => new GitHubRemoteRepository('googleapis/gax-php', $gaxRootDir),
-            'default_opened_level' => 1,
```

Comment:
Why has this been removed?

Reply:
The API render was odd, default_opened_level = 2 just looks right
should I revert ?